client/asset/dcr/dcr.go

```diff
@@ -561,9 +561,19 @@ func (dcr *ExchangeWallet) Info() *asset.WalletInfo {
 	return WalletInfo
 }
 
+// var logup uint32
+
+// func rpclog(log dex.Logger) {
+// 	if atomic.CompareAndSwapUint32(&logup, 0, 1) {
+// 		rpcclient.UseLogger(log)
+// 	}
+// }
+
 // Connect connects the wallet to the RPC server. Satisfies the dex.Connector
-// interface.
+// interface. WARNING: Once stopped, it cannot reconnect, requiring NewWallet to
+// construct a new ExchangeWallet to Connect again.
```

Comment:
We could consider allowing this, but it would require creating a new rpcclient.Client on the fly, which would require storing the config and detecting if the client had been previously shutdown.  I'm inclined to document proper use here instead.